CHANGELOG.md

```diff
@@ -20,9 +20,9 @@ A brief description of the categories of changes:
 * Particular sub-systems are identified using parentheses, e.g. `(bzlmod)` or
   `(docs)`.
 
-## Unreleased
+## [0.37.0] - 2024-10-18
```

Comment:
Having the blocks added is a nice to have here.

```suggestion
## Unreleased

### Changed
- Nothing yet

### Fixed
- Nothing yet

### Added
- Nothing yet

### Removed
- Nothing yet

## [0.37.0] - 2024-10-18
```

Reply:
Done. Thanks, I was a bit lazy :sweat_smile: